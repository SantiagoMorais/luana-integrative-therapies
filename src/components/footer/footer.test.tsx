import { render, screen } from "@testing-library/react";
import { Footer } from "./index";

describe("<Footer />", () => {
    it('should render the component correctly', () => {
        render(<Footer />);
        const title = screen.getByText(/Me encontre nas minhas mÃ­dias sociais/i)
        expect(title).toBeInTheDocument();
    })

    it('should the buttons have the correct url to the corresponding button', () => {
        render(<Footer />);
        const social = screen.getByRole("list");
        expect(social).toMatchSnapshot();
    });
});