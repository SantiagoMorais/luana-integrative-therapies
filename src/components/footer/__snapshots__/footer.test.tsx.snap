// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Footer /> > should the buttons have the correct url to the corresponding button 1`] = `
<ul
  class="navigation"
>
  <a
    class="section"
    href="/"
  >
    home
  </a>
  <a
    class="section"
    href="/about"
  >
    sobre mim
  </a>
  <a
    class="section"
    href="/contact"
  >
    contato
  </a>
  <a
    class="section"
    href="/coming-soon"
  >
    equilibrium
  </a>
  <a
    class="section"
    href="/coming-soon"
  >
    segredos da lua
  </a>
</ul>
`;
